lms/static/js/student_account/views/FormView.js

```diff
@@ -1,270 +1,267 @@
-var edx = edx || {};
-
-(function($, _, Backbone, gettext) {
+;(function (define) {
     'use strict';
+    define(['jquery', 'underscore', 'backbone', 'js/utils/edx.utils.validate'],
+        function($, _, Backbone, EdxUtilsValidate) {
 
-    edx.student = edx.student || {};
-    edx.student.account = edx.student.account || {};
-
-    edx.student.account.FormView = Backbone.View.extend({
-        tagName: 'form',
+        var FormView = Backbone.View.extend({
```

Comment:
Instead of assigning the view to a variable and returning that you can just return the viewl, i.e.
```
return Backbone.View.extend({
```